src/com/sap/piper/CfUtils.groovy

```diff
@@ -0,0 +1,40 @@
+package com.sap.piper
```

Comment:
This should go into the `integration` package and be implemented as a class, see [TransportManagementService](https://github.com/SAP/jenkins-library/blob/master/src/com/sap/piper/integration/TransportManagementService.groovy) as an example.